src/components/inputs/markdown/ohri-markdown.component.tsx

```diff
@@ -2,7 +2,15 @@ import React from 'react';
 import ReactMarkdown from 'react-markdown';
 import { OHRIFormFieldProps } from '../../../api/types';
 
-const OHRIMarkdown: React.FC<OHRIFormFieldProps> = ({ question }) =>
-  !question.isHidden && <ReactMarkdown children={question.value.join('\n')} />;
-
+const OHRIMarkdown: React.FC<OHRIFormFieldProps> = ({ question }) =>{
+  if (!question.isHidden) {
+    if (typeof question.value === 'string') {
+      return <ReactMarkdown>{question.value}</ReactMarkdown>;
+    } else if (Array.isArray(question.value)) {
+      return <ReactMarkdown children={question.value.join('\n')} />;
+    } else {
+      return <div>Invalid value type to render</div>;
+    }
+  }
```

Comment:
@reagan-meant I would simply:

```jsx
const markdownValue = Array.isArray(question.value) ? question.value.join('\n') : question.value;
return !question.isHidden && <ReactMarkdown>{markdownValue}</ReactMarkdown>;
```

Two more issues:

1. Did we agree upon supporting a limited number of elements?
2. You seem to have a formatting issue on line 5